rest-api-spec/src/main/resources/rest-api-spec/test/indices.forcemerge/20_wait_for_completion.yml

```diff
@@ -4,15 +4,17 @@
   # will return a task immediately and the merge process will run in background.
 
   - skip:
-      version: " - 2.6.99"
-      reason: "only available in 2.7+"
-      features: allowed_warnings
+      version: " - 2.6.99, 2.13.0 - "
+      reason: "wait_for_completion was introduced in in 2.7.0 and task description was changed in 2.13.0"
```

Comment:
there is redundant in in the message `... was introduced in in 2.7.0 ... `

Reply:
Good catch, thank you, I've changed that.